db/migrations/20260820_0049_marketplace_guest_quotes.sql

```
@@ -0,0 +1,487 @@
+      add constraint marketplace_quote_invitations_identity_unique
+      unique (id, quote_request_id, profile_id);
+  end if;
+end;
+$$;
+
+create index if not exists marketplace_quote_invitations_quote_idx
+  on marketplace_quote_invitations (quote_request_id, status, created_at desc);
+create index if not exists marketplace_quote_invitations_profile_idx
+  on marketplace_quote_invitations (profile_id, status, created_at desc);
+create index if not exists marketplace_quote_invitations_recipient_idx
+  on marketplace_quote_invitations (lower(recipient_email), status, created_at desc);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)